Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,65 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        // GUI model loading
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
+        Vec3 offset = new Vec3(0.11, 0.22, 0.33);
+        Body body = model.getBodySet().get(0);
+        String newMarkerName = "newMarker";
+        Marker newMarker = new Marker();
+        newMarker.setName(newMarkerName);
+        newMarker.set_location(offset);
+        newMarker.setParentFrameName(body.getName());
+        context.cacheModelAndState();
+        model.addMarker(newMarker);
+        try {
+            context.restoreStateFromCachedModel();
```

Comment:
Is this try-catch sufficient to ensure that adding the marker actually worked? Similar comment for the rest of the test.

Reply:
The catch is intended for situation where trying to restore the state (e.g. Coordinate values etc.) fails. I don't expect this exception to be fired here when adding a Marker doesn't interact with the state but may happen in other contexts.